scripts/secure-boot/generate-images/intermediate.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# go into a separate directory to make cleanup easier
+mkdir -p something
+cd something
+
+# make the new image
+dd if=/dev/zero of=image.img bs=1M count=8
```

Comment:
@WiktorG351 count could be a variable, please use variables instead of hardcoded values